### ISSUE TITLE
RN build failing because of a missing intellij-core library

### ISSUE THREAD

**Issue by plozachm** [naoufal/react-native-touch-id#164]
Hi,

react-native-touch-id: 4.1.0   [PROBLEM SOLVED IN 4.1.1]
RN version: 0.57.3

Notice: Same exact configuration was working yesterday (my colleagues with the jar in cache can still build the app)

Since this morning my build are failing because of a missing jar on jcenter.bintray.com (the pom file is there but nor the jar file):

```
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring project ':react-native-touch-id'.
> Could not resolve all artifacts for configuration ':react-native-touch-id:classpath'.
   > Could not find intellij-core.jar (com.android.tools.external.com-intellij:intellij-core:26.0.1).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/external/com-intellij/intellij-core/26.0.1/intellij-core-26.0.1.jar
```
Debug log attached:
[build_debug.log](https://github.com/naoufal/react-native-touch-id/files/2506434/build_debug.log)

To be confirm  but it might be related to the version of gradle
https://github.com/naoufal/react-native-touch-id/blob/master/android/build.gradle

The root cause does not look related to this react-native-touch-id but is there anything you could do?
Thanks in advance for your help!

P.S.:
https://stackoverflow.com/questions/52946058/android-build-miss-intellij
https://stackoverflow.com/questions/52946371/android-studio-could-not-find-intellij-core-jar

**Comment by van1985** [comment]
The same problem here....any update how can I fix this issue? Thanks!

**Comment by van1985** [comment]
One comment more, if in the build.gradle from react-native-touch-id project change the order works.

From this:

buildscript {
  repositories {
    jcenter()
    maven {
      url 'https://maven.google.com/'
      name 'Google'
    }
  }


to this...

buildscript {
  repositories {
           maven {
      url 'https://maven.google.com/'
      name 'Google'
    }
       jcenter()
  }

**Comment by zibs** [comment]
Does changing the jcenter import actually fix it? I'm not able to do any testing until later this week - but if a few people chime in and say that it's working and not breaking things, I can merge the PR that has this in it

**Comment by van1985** [comment]
I tried that @AndriiChubariev and not works in my CI. Still failing.

**Comment by van1985** [comment]
That fix for me the CI, the same happens for react-native-camera. I pull down jcenter().

**Comment by van1985** [comment]
react-native-camera put the fix that @AndriiChubariev said: https://github.com/react-native-community/react-native-camera/commit/9261524609ac4d9f29a33fb94847ef78a50b38f3

**Comment by AndriiChubariev** [comment]
@van1985 
To make it work in CI(at least today, while the new version is not rolled out in npm), you can update your package.json
from:
```
...
dependencies: {
...
    "react-native-camera": "^1.3.0",
    "react-native-touch-id": "^4.0.4",
...
```
to:
```
...
dependencies: {
...
    "react-native-camera": "react-native-community/react-native-camera",
    "react-native-touch-id": "naoufal/react-native-touch-id",
...
}

This will point dependencies from NPM to Github repo, master branch directly.
But this requires https://github.com/naoufal/react-native-touch-id/pull/166 to be merged first.

**Comment by sypl** [comment]
#166 has been merged, but it does not seem to have fixed the issue for us in CI.

**Comment by jamesmcn1** [comment]
Can confirm new package version 4.1.0 builds successfully on local and CI.

**Comment by sypl** [comment]
Apologies, this fix is actually probably good. My bad, our dependency was react-native-camera, not react-native-touch-id. Both were suffering from the same issue.

**Comment by van1985** [comment]
Excellent @jamesmcn1 !! Thanks for the quick response =)

**Comment by zibs** [comment]
This is fixed in 4.2.0 and 4.3.0
